### PR TITLE
Improve code base readability

### PR DESCRIPTION
**Pull request by swarajgiri**
### Changes contain
- Improve consistency of vertical spacing
- Minimize `else` and `elseif` conditionals 
- Remove variables created just for `return` statements
- Use `===` instead of `==`

### Test cases affected
- None

### CODE REVIEW

**Comment by nyamsprod** [comment]
Hi @swarajgiri Thanks for the work but I'm currently reviewing and making many changes to the v4 version to match #57 so many changes will be committed soon that may conflict or resolve your pull request. I would prefer you wait a little longer so that you contribution won't be crash by my own commit.

I'll keep in your changes in mind before committing the new code.

File: src/AbstractComponent.php [hunk 1]
```diff
@@ -78,6 +84,6 @@ public function __toString()
      */
     public function sameValueAs(ComponentInterface $component)
     {
-        return $this->__toString() == $component->__toString();
```
Comment (nyamsprod) [inline src/AbstractComponent.php]: the "===" is not needed as we know that we are comparing string so it won't add or make any difference there

Comment (swarajgiri) [inline src/AbstractComponent.php]: @nyamsprod - Was trying to be consistent :D

**Comment by swarajgiri** [comment]
@nyamsprod - Yeah, sure. No hurry. 
Give me a shout when you are done with changes for v4 and i will rebase.

**Comment by nyamsprod** [comment]
The merge has been made I'll close this pull request

**Pull request closed without merging**